opcua/server/server.py

```diff
@@ -416,3 +416,6 @@ def subscribe_server_callback(self, event, handle):
 
     def unsubscribe_server_callback(self, event, handle):
         self.iserver.unsubscribe_server_callback(event, handle)
+
+    def link_method(self, node, callback):
+        node.server.add_method_callback(node.nodeid, callback)
```

Comment:
Are you sure your idea will work?

The `add_method_callback` method is part of InternalSession, not InternalServer.